procurement_portal/records/views.py

```diff
@@ -17,6 +20,22 @@
 )
 
 
+class State:
```

Comment:
I wouldn't call this generically State. How about PurchaseRecordFieldsState or something? then we can just call the property `fields`